modules/homepage-slider/homepage-slider.model.js

```diff
@@ -15,6 +15,10 @@ const HomePageSlideSchema = new mongoose.Schema({
   images: ImageSet,
   order: Number,
   show: Boolean,
+  translations_key: {
```

Comment:
!!!!!!!!!!!!!!!!!!!!!!CAMELCASE!!!!!!!!!!!!!!!!!!!!!!!!!

Reply:
Sorry, but for now we should use this approach and after everyone will complete similar tasks, we are gonna add "translations" camelCase fix